salt/modules/bamboohr.py

```diff
@@ -32,7 +32,7 @@ def __virtual__():
     '''
     if _apikey():
         return True
-    return False
+    return (False, 'The API key was not specified. Please specify it using the "apiKey" attribute.')
```

Comment:
@brendan-rius, this should be `"apikey"`, and it might be less confusing if it were called the `"apikey" config`.